utils/helpers.go

```diff
@@ -279,6 +279,9 @@ func WriteAzionJsonContent(conf *contracts.AzionApplicationOptions) error {
 
 // Returns the correct error message for each HTTP Status code
 func ErrorPerStatusCode(httpResp *http.Response, err error) error {
+	//if httpResp != nil {
```

Comment:
Do we need these commented lines? 🤔

Reply:
At the moment it's not an idea, to minimize code but it affects many tests, so I'll need to send it in a separate pr